sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -247,12 +248,41 @@ export default {
         event.changed?.component.key === OFFERINGS_DROPDOWN_KEY &&
         +event.changed.value > 0
       ) {
-        await formioComponentLoader.loadSelectedOfferingDate(
+        await formioComponentLoader.loadSelectedOfferingDetails(
           form,
           +event.changed.value,
-          SELECTED_OFFERING_DATE_KEY,
+          {
+            offeringEndDateFieldId: SELECTED_OFFERING_END_DATE_KEY,
```

Comment:
Can we have the parameters ordered as start and then end dates?

Reply:
Offering start date is optional as it is used only in student application form and optional value must have to come after the mandatory one.